### ISSUE TITLE
Documentation Progress

### ISSUE THREAD

**Issue by Tropix126** [Tropix126/fluent-svelte#13]
### Components
- [ ] AutoSuggestBox
- [x] Button
- [x] Checkbox
- [ ] ComboBox
- [x] ContentDialog
- [ ] ContextMenu
- [x] Expander
- [ ] Flyout
- [ ] IconButton
- [ ] InfoBar
- [ ] ListItem
- [ ] MenuBar
- [ ] MenuBarItem
- [ ] MenuFlyout
- [ ] MenuFlyoutItem
- [ ] NumberBox
- [ ] PersonPicture
- [ ] ProgressBar
- [ ] ProgressRing
- [ ] RadioButton
- [x] Slider
- [ ] TextBlock
- [ ] TextBox
- [ ] ToggleSwitch
- [ ] Tooltip

### Internals
- [ ] MenuFlyoutSurface
- [ ] FlyoutSurface
- [ ] TooltipSurface
- [ ] ComboBoxItem
- [ ] `externalMouseEvents`
- [ ] `focusTrap`
- [ ] `uid`
- [ ] `arrowNavigation`
- [ ] `getCSSDuration`
- [ ] `createEventForwarder`

**Comment by rgossiaux** [comment]
Poking around your site, I noticed that when you're on the Flyout page you can't navigate anywhere else (the links don't work) while the Flyout is open, which is kind of confusing.

**Comment by Tropix126** [comment]
> Poking around your site, I noticed that when you're on the Flyout page you can't navigate anywhere else (the links don't work) while the Flyout is open, which is kind of confusing.

Yep. This should be fixed as of last evening. Was caused by `focus-trap` causing trouble.

**Comment by achilesmatheus** [comment]
Is this documentation still being updated?

**Comment by fox62843** [comment]
Has this project stopped? I feel it is too much of a waste.

It's sad, because I'm excited about the overwhelming degree of completion, which is as good as the native version. I really want you to finish it and release it to the world.
